db/structure.sql

```diff
@@ -2,12 +2,16 @@
 -- PostgreSQL database dump
```

Comment:
What is this file for? I've never seen this. Is this instead of a schema.rb or something?